topics/ratchet/index.md

```diff
@@ -0,0 +1,12 @@
+---
+related: php, websockets
```

Comment:
How about also "library"? Just thinking if someone very new to programming comes here, https://github.com/topics/library has an explanation of what a library is.

Could also list `client` and `server`.